tools/diffbind/diffbind.xml

```diff
@@ -3,6 +3,7 @@
     <requirements>
         <requirement type="binary">Rscript</requirement>
         <requirement type="R-module">DESeq2</requirement>
+        <requirement type="package" version="1.2.8">zlib</requirement>
```

Comment:
@martenson Yes, this can create problems in other resolvers (e.g. Conda), where diffbind may require a different zlib version.